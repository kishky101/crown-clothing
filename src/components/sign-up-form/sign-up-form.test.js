import renderer from 'react-test-renderer';
import SignUp from './sign-up-form.component';
import { MockStore } from '../../utils/test/setup';

test('creating sign-up-form', () => {
    const signup = renderer.create(
        <MockStore>
            <SignUp />
        </MockStore>
    ).toJSON;
    expect(signup).toMatchSnapshot();
})