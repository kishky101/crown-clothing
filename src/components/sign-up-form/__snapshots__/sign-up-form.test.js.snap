// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`creating sign-up-form 1`] = `[Function]`;
